libs/execution_context/ebpf_program.c

```diff
@@ -431,11 +431,11 @@ _ebpf_program_type_specific_program_information_attach_provider(
         goto Done;
     }
 
-    if (program_data->required_irql > HIGH_LEVEL) {
+    if (program_data->maximum_irql > HIGH_LEVEL) {
```

Comment:
Should `minimum_irql` should be similarly constrained, or at least constrained to be `<= maximum_irql` ?